tests/test-ocfserver.js

```diff
@@ -5,6 +5,9 @@ console.log("Test OCF server");
 var ocf = require("ocf");
 var assert = require("Assert.js");
 
+var bleAddress = "F1:E2:D3:C4:B5:A6";
+ocf.setBleAddress(bleAddress);
+
```

Comment:
Note that this will only work on the A101, on the FRDM board it will likely throw an error. The (IMO better) way to set the BLE address is via the command line with:
`BLE_ADDR="<address>"`

But if the only test target is arduino 101, then this is fine.